[Bug][Teambition] Swagger generation fails: cannot find type definition models.TeambitionRepo

### Search before asking

- [x] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.


### What happened

When attempting to build DevLake with the teambition plugin enabled, the Swagger documentation generation step (swag init) fails due to a missing type reference in the file scope_api.go.

Specifically, the comment annotations reference models.TeambitionRepo, but the type is either undefined or not properly exported/imported. This causes swag to fail and abort the make all process, preventing successful Docker builds or local development.

Having the following error while in the CI also:
```
209.8 2025/06/25 07:18:45 ParseComment error in file /app/plugins/teambition/api/scope_api.go :cannot find type definition: models.teambitionRepo
209.8 make: *** [Makefile:46: swag] Error 1
------
WARNING: No output specified with docker-container driver. Build result will only remain in the build cache. To push result image into registry use --push or to load image into docker use --load
Dockerfile:93
```

### What do you expect to happen

The CI should correctly build.

### How to reproduce

1. Set GO_PLUGINS=teambition
2. Run make all (or make swag)
3. Observe the error related to missing type definition during Swagger generation

### Anything else

happening since:
691b14f01750246db097b3c0466c9a34d470148a

### Version

691b14f01750246db097b3c0466c9a34d470148a

### Are you willing to submit PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/backend/plugins/teambition/api/scope_api.go
+++ b/backend/plugins/teambition/api/scope_api.go
@@ -34,7 +34,7 @@ type ScopeDetail api.ScopeDetail[models.TeambitionProject, models.TeambitionScop
 // @Accept application/json
 // @Param connectionId path int true "connection ID"
 // @Param scope body PutScopesReqBody true "json"
-// @Success 200  {object} []models.teambitionRepo
+// @Success 200  {object} []models.TeambitionProject
 // @Failure 400  {object} shared.ApiBody "Bad Request"
 // @Failure 500  {object} shared.ApiBody "Internal Error"
 // @Router /plugins/teambition/connections/{connectionId}/scopes [PUT]
@@ -49,8 +49,8 @@ func PutScopes(input *plugin.ApiResourceInput) (*plugin.ApiResourceOutput, error
 // @Accept application/json
 // @Param connectionId path int true "connection ID"
 // @Param scopeId path int true "scope ID"
-// @Param scope body models.teambitionRepo true "json"
-// @Success 200  {object} models.teambitionRepo
+// @Param scope body models.TeambitionProject true "json"
+// @Success 200  {object} models.TeambitionProject
 // @Failure 400  {object} shared.ApiBody "Bad Request"
 // @Failure 500  {object} shared.ApiBody "Internal Error"
 // @Router /plugins/teambition/connections/{connectionId}/scopes/{scopeId} [PATCH]
@@ -97,7 +97,7 @@ func GetScope(input *plugin.ApiResourceInput) (*plugin.ApiResourceOutput, errors
 // @Param connectionId path int true "connection ID"
 // @Param scopeId path int true "scope ID"
 // @Param delete_data_only query bool false "Only delete the scope data, not the scope itself"
-// @Success 200  {object} models.teambitionRepo
+// @Success 200  {object} models.TeambitionProject
 // @Failure 400  {object} shared.ApiBody "Bad Request"
 // @Failure 409  {object} srvhelper.DsRefs "References exist to this scope"
 // @Failure 500  {object} shared.ApiBody "Internal Error"

